ambry-store/src/test/java/com.github.ambry.store/LogTest.java

```diff
@@ -567,6 +582,11 @@ private void checkLog(Log log, long expectedSegmentCapacity, List<String> expect
       }
     }
     assertNull("Next segment should be null", nextSegment);
+    assertEquals("Unexpected result for isLogSegment", log.getCapacityInBytes() != expectedSegmentCapacity,
+        log.isLogSegmented());
+    long expectedSegmentsNeeded = (log.getCapacityInBytes() / expectedSegmentCapacity) - expectedSegmentNames.size();
+    assertEquals("DiskSpaceRequirements has wrong number of unallocated segments", expectedSegmentsNeeded,
```

Comment:
message seems wrong

Reply:
This is actually the number of unallocated segments that it is checking. I will change that variable name instead